broker/amqp.go

```diff
@@ -5,19 +5,28 @@ import (
 	"fmt"
 	"strconv"
 
+	"github.com/google/uuid"
 	"github.com/streadway/amqp"
 )
 
 // ErrDisconnected occurs when trying to do something that requires a connection but one was
 // unavailable
 var ErrDisconnected = errors.New("disconnected from the broker")
 
+// ErrorNoRes occurs when no response is returned from the server on an RPC call
+var ErrNoRes = errors.New("no response from server")
+
+// ErrRpcQueueAssertionFailure occurrs when the anon RPC queue fails to create
+var ErrRpcQueueAssertionFailure = errors.New("failed to create anonymous rpc queue")
+
 // AMQP is a broker for AMQP clients. Probably most useful for RabbitMQ.
 type AMQP struct {
 	conn            *amqp.Connection
 	channel         *amqp.Channel
 	receiveCallback EventHandler
 	consumerTags    map[string]string
+	rpcqueue        amqp.Queue
+	rpcconsumer     <-chan amqp.Delivery
```

Comment:
Can we make these camel case to remain consistent with convention and other struct fields?